RSS Reader/RSS Reader/Default/AppDelegate.m

```diff
@@ -0,0 +1,61 @@
+//
+//  AppDelegate.m
+//  RSS Reader
+//
+//  Created by Lina Loyko on 11/17/20.
+//
+
+#import "AppDelegate.h"
+#import "RSSRTopicsListViewController.h"
+#import "RSSRNetworkService.h"
+#import "RSSRXMLParser.h"
+#import "RSSRTopicsListPresenter.h"
+
+@interface AppDelegate ()
+
+@end
+
+@implementation AppDelegate
+
+
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    if (@available(iOS 13.0, *)) {
+        
+    } else {
+        UIWindow *window = [[UIWindow alloc] initWithFrame:UIScreen.mainScreen.bounds];
+        self.window = window;
```

Comment:
use autorelease to reduce the amount of local variables created

Reply:
Done